doc/src/pair_sw.rst

```diff
@@ -111,6 +119,9 @@ This value enables the cut-off function to exclude unnecessary angles in the thr
    However, the angle variation is much smaller than the given threshold value for actual simulations,
    so the inconsistency between potential and force can be neglected in actual simulations.
 
+The *threebody* keyword determines whether or not the three-body term of the potential is calculated. Skipping this
+significantly increases the speed of the potential, but is only applicable when :math:`\lambda_{ijk} = 0`
```

Comment:
> "and is equivalent to setting :math:`\lambda_{ijk}` to zero".

I feel like this implies that you get the same speed increase from setting lambda to zero as you would from using the keyword. 

I think I can sidestep this while knocking out the issue with the derived potentials you mentioned. I can set lambda to zero if `threebody_flag` is false, which should standardize the expected interaction across the derived classes. Although a it should be documented that `threebody on/off` won't work for sw/mod because it overrides `settings()`.

With that in place, I'd be able to write something like: "The *threebody* keyword determines whether or not the three-body term of the potential is calculated. Skipping this significantly increases the speed of the calculation, with the tradeoff that :math:`\lambda_{ijk}` is forcibly set to 0. If the keyword is used with the pair styles sw/angle/table, sw/gpu, sw/intel, or sw/kokkos, :math:`\lambda_{ijk}` will be forcibly set to 0, but no benefits will be gained. This keyword cannot be used for variants of sw/mod."

Reply:
This is partially solved by 419460f and d7c4a49. 

There is a weird interaction I'm not sure how I want to handle. sw/angle/table has its own read_file() function, which means it doesn't inherit my forcing lambda to be zero when the threebody flag is set. sw/angle/table does inherit the settings() function, however. 

This means that sw/angle/table will accept the threebody flag, but do nothing with it. I'm not sure how this should be handled, if I should just document it and call it a day, make the change to the sw/angle/table style to force lambda to be zero for consistency, or override the base sw settings function to reject the threebody keyword.

Reply:
It makes no sense to turn threebody interactions off for pair style sw/angle/table since the whole point of the pair style is to *replace* the threebody term with a tabulated function (so changing the lambda values has no effect).

Reply:
I have "massaged" the documentation a bit more to make it more succinct. Please check it out.